packages/cashier/src/containers/routes/__tests__/route-with-sub-routes.spec.tsx

```diff
@@ -1,48 +1,48 @@
-// TODO refactor old tests in this component
 import React from 'react';
-import { RouteWithSubRoutesRender } from '../route-with-sub-routes';
-import { MemoryRouter, Redirect } from 'react-router-dom';
-import { render } from '@testing-library/react';
+import { Redirect } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+import RouteWithSubRoutes from '../route-with-sub-routes';
+
+type TMockFunction = {
+    path: string;
+    exact?: boolean;
+};
+
+jest.mock('react-router-dom', () => ({
+    ...jest.requireActual('react-router-dom'),
+    Route: jest.fn(({ path, exact }: TMockFunction) => (
+        <div>
+            <span>{`path param: ${path}`}</span>
+            <span>{`exact param: ${exact}`}</span>
+        </div>
+    )),
+}));
+
+afterEach(() => jest.clearAllMocks());
 
-const mockFunction = jest.fn();
 const route = {
-    getTitle: mockFunction,
+    getTitle: jest.fn(),
     component: Redirect,
     is_logging_in: true,
     is_logged_in: true,
     exact: true,
-    path: '/',
-    to: '/root',
+    path: '/test-path',
 };
 
-const MockComponent = () => (
-    <MemoryRouter>
-        <RouteWithSubRoutesRender {...route} />
-    </MemoryRouter>
-);
+const MockRouteWithSubRoutes = () => <RouteWithSubRoutes {...route} />;
```

Comment:
@niloo-fs what do you think about using <RouteWithSubRoutes {...route} /> directly? 🙂 this line feels extra to me